pkg/domain/infra/abi/containers.go

```diff
@@ -838,6 +844,11 @@ func (ic *ContainerEngine) ContainerStart(ctx context.Context, namesOrIds []stri
 				}
 				report.Err = errors.Wrapf(err, "unable to start container %q", ctr.ID())
 				reports = append(reports, report)
+				if ctr.AutoRemove() {
+					if err := ic.removeContainer(ctx, ctr, entities.RmOptions{}); err != nil {
```

Comment:
just thinking, should we default to `--ignore` in the `RmOptions{}` since either way we log that the container isn't in existence? Maybe change the log to explicitly say the container doesn't exist but ignoring.

Reply:
SGTM

Reply:
`--ignore` is useful when doing  a `podman container rm --ignore notSureExists`.  But in this case, we know that the container exists. So I think that setting `RmOptions.Ignore = true` may suppress otherwise valid errors - in this specific case.